apps/coscrad-frontend/src/components/resources/utils/render-aggregate-id-cell.tsx

```diff
@@ -3,5 +3,7 @@ import { Link } from 'react-router-dom';
 import { CellRenderer } from '../../../utils/generic-components/presenters/tables/generic-index-table-presenter/types';
 
 export const renderAggregateIdCell: CellRenderer<HasId> = ({ id }: HasId) => (
-    <Link to={id}>VIEW</Link>
+    <Link data-testid={id} to={id}>
```

Comment:
Do we need a `data-testid` or can we query for an actual link with the selector?

Reply:
I tried to get this and the issue in photographs is that the thumbnail image is also hyperlinked so it's got a collection of two instead of the one "View" link.  If we were to change to an icon or button this would shift too.

Reply:
```ts
    /**
     * We would prefer to simply look for the link instead of a test ID. The issue,
     * e.g., in Photographs is that there's both the VIEW link and the thumbnail image is also hyperlinked so that
     * ```ts
     * cy.get([href="/Resources/Photographs/${photographId}"]).click();
     * ```
     * retrieves both links and cypress complains. If we go with a Button,
     * we could use that as an additional criteria to locate the right link,
     * but that would create problems if we later got rid of the button.
     */
    <Link data-testid={id} to={id}>
        <Button variant="outlined">VIEW</Button>
    </Link>
```